doc/vim-go.txt

```diff
@@ -566,13 +566,13 @@ CTRL-t
                                                                  *:GoFreevars*
 :GoFreevars
 
-    Enumerates the free variables of the selection. “Free variables” is a
+    Enumerates the free variables of the selection. "Free variables" is a
```

Comment:
You're not smart quote hip :-(

Reply:
I'm hip, but vimhelplint wasn't in Linux without some particular encodings installed.